docs/examples/imageset-config-release.yaml

```diff
@@ -1,10 +1,12 @@
+# This config demonstrates how to mirror a specified
+# version(s) of openshift. Optionally, omit the version to mirror
+# the latest release.
+---
 apiVersion: mirror.openshift.io/v1alpha1
 kind: ImageSetConfiguration
 mirror:
   ocp:
     channels:
-      - name: stable-4.8
-      - name: stable-4.7
+      - name: stable-4.9
         versions:
-          - 4.7.36
-    graph: true
\ No newline at end of file
+        - 4.9.13
```

Comment:
I think this indentation is not correct, is it?

Reply:
Also, the `graph: true` is removed here. I assume the field is deprecated/removed?

Reply:
Weird, the indentation works either way in testing, but I think that you are right about the syntax being technically incorrect as written here.

Reply:
We do not use the graph key yet.